README.md

```diff
@@ -29,6 +29,10 @@ Setup
 rake db:setup # will create, migrate and seed the db
 ```
 
+Note- it's generally encouraged to develop with VSCode, and take advantage of some of the extensions. These include GraphQL syntax highlighting, autocomplete, cmd + click to see the definition, etc.
+
+Everything should just work when using VSCode with the recommended extensions, as well as running `rake graphql:dump_schema`. This will write Constellation's GraphQL schema to disk, powering the VSCode extensions.
```

Comment:
This is missing that they need to [install watchman](https://facebook.github.io/watchman/docs/install.html) and run `npm install @playlyfe/gql` to get the autocompletion, as per [the docs](https://github.com/kumarharsh/graphql-for-vscode#setting-it-up).

(This should go away if https://github.com/kumarharsh/graphql-for-vscode/issues/42 gets addressed.)

Reply:
Oh right, yes!